tools/tune_inter_intra.py

```diff
@@ -0,0 +1,75 @@
+import multiprocessing
+import sys
+import itertools
+import subprocess
+import os
+import argparse
+
+def eprint(*args, **kwargs):
+    print(*args, file=sys.stderr, **kwargs)
+
+def csv_print(*kwargs):
+    print(*kwargs, sep=',')
+
+N_PHYSICAL_CORES = multiprocessing.cpu_count() // 2
```

Comment:
Could it be interesting to include all logical cores in the benchmark? Or do you think there is not much to gain from hyperthreading?

Reply:
I think that it is unlikely that it would make any difference, it will always slow down the tuning and it would require really big input test files to avoid work imbalance.